unlock-app/src/components/content/VerificationContent.tsx

```diff
@@ -1,26 +1,48 @@
 import Head from 'next/head'
 import { useRouter } from 'next/router'
-import React, { useState } from 'react'
+import React, { useEffect, useState } from 'react'
+import { useAuth } from '~/contexts/AuthenticationContext'
+import { getMembershipVerificationConfig } from '~/utils/verification'
+import { useStorageService } from '~/utils/withStorageService'
+import { useWalletService } from '~/utils/withWalletService'
 import { pageTitle } from '../../constants'
 import LocksContext from '../../contexts/LocksContext'
 import Account from '../interface/Account'
 import Layout from '../interface/Layout'
-import Loading from '../interface/Loading'
+import { Scanner } from '../interface/verification/Scanner'
 import VerificationStatus from '../interface/VerificationStatus'
 
 export const VerificationContent: React.FC<unknown> = () => {
   const { query } = useRouter()
   const [locks, setLocks] = useState({})
-  let data
-  let sig
+  const storageService = useStorageService()
+  const walletService = useWalletService()
+  const { account, network } = useAuth()
 
-  if (typeof query.data === 'string' && typeof query.sig === 'string') {
-    data = decodeURIComponent(query.data)
-    sig = query.sig
-  }
+  useEffect(() => {
+    if (account && network && walletService) {
+      storageService.loginPrompt({
```

Comment:
I really want to make sure we have a good logged out experience so people can scan tickets too even if they don't have wallets.

Reply:
They can but they won't be able to check in.